### PR TITLE
fix CalabiYau Serialize SkeletalMesh error

### PR DESCRIPTION
**Pull request by zhuyule**

### CODE REVIEW

File: CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkelMeshSection.cs [hunk 1]
```diff
@@ -268,7 +268,7 @@ public void SerializeRenderItem(FAssetArchive Ar)
                 Ar.Position += 4;
                 break;
             case EGame.GAME_CalabiYau or EGame.GAME_FragPunk:
-                Ar.Position += 8;
+                Ar.Position += 14;
```
Comment (GMatrixGames) [inline CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkelMeshSection.cs:271]: This needs to be validated to ensure compatibility with FragPunk as well

File: CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkeletalMaterial.cs [hunk 2]
```diff
@@ -44,7 +44,7 @@ public FSkeletalMaterial(FAssetArchive Ar)
             if (FRenderingObjectVersion.Get(Ar) >= FRenderingObjectVersion.Type.TextureStreamingMeshUVChannelData)
                 UVChannelData = new FMeshUVChannelInfo(Ar);
 
-            if (Ar.Game is EGame.GAME_CalabiYau or EGame.GAME_FragPunk) Ar.Position += 4;
+            if (Ar.Game is EGame.GAME_CalabiYau or EGame.GAME_FragPunk) Ar.Position += 8;
```
Comment (GMatrixGames) [inline CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkeletalMaterial.cs:47]: This needs to be validated to ensure compatibility with FragPunk

**Review by GMatrixGames — CHANGES_REQUESTED** [review]
I don't deal with mesh export, but the main code changes need to be validated with FragPunk as well as it's part of the conditional. If it's not compatible, they need to be split into two statements.

File: CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkelMeshSection.cs (hunk 1 above)
Comment (zhuyule) [reply CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkelMeshSection.cs:276]: sorry, i do not have FragPunk ,so i split the change to two statements

**Pull request closed without merging**